content/S2016/S16-event-office-move.md

```diff
@@ -0,0 +1,11 @@
+Title: Our office has moved!
+Date: 2016-8-15
+Category: Events
```

Comment:
I agree, but I was under the impression that the category determined what page it showed up on. @fboxwala probably knows more about this